components/desktop/freerdp/manifests/sample-manifest.p5m

```diff
@@ -10,7 +10,7 @@
 #
 
 #
-# Copyright 2020 <contributor>
+# Copyright 2021 <contributor>
 #
```

Comment:
Please remove the sample-manifest from the PR. It's only needed if something has changed. Change of the copyright year is not sufficient.